app/views/directives/edit-webhook-triggers.html

```diff
@@ -0,0 +1,30 @@
+<h5>{{type}} webhooks
+  <span class="help action-inline">
+    <a href>
+      <i class="pficon pficon-help" data-toggle="tooltip" aria-hidden="true" data-original-title="{{typeInfo}}">
```

Comment:
I'd also put the aria-hidden on the parent `a` element. Otherwise screen readers will read it as an empty link.